k8s/build-docker-image-from-bin/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:20.04
+
+ENV PATH="$PATH:/var/apache-zookeeper-3.7.1-bin/bin:/var/RaftKeeper/bin"
+ENV RAFTKEEPER_DIR="/var/RaftKeeper"
+
+EXPOSE 8101 8102 8103
+
+# For Chainese users, if there is something wrong, just comment it.
+RUN echo "# use mirrors" > /etc/apt/sources.list \
+    && echo "deb [trusted=yes] http://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal main restricted universe multiverse" >> /etc/apt/sources.list \
+    && echo "deb [trusted=yes] http://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal-updates main restricted universe multiverse" >> /etc/apt/sources.list \
+    && echo "deb [trusted=yes] http://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal-backports main restricted universe multiverse" >> /etc/apt/sources.list \
+    && echo "deb [trusted=yes] http://mirrors.tuna.tsinghua.edu.cn/ubuntu/ focal-security main restricted universe multiverse" >> /etc/apt/sources.list \
+    && apt-get update \
+    && apt-get install wget openjdk-8-jre-headless python3 netcat --yes \
+    && cd /var && wget https://dlcdn.apache.org/zookeeper/zookeeper-3.7.1/apache-zookeeper-3.7.1-bin.tar.gz && tar -xzf apache-zookeeper-3.7.1-bin.tar.gz && rm apache-zookeeper-3.7.1-bin.tar.gz \
+    && wget https://github.com/Hooollin/RaftKeeper/releases/download/1.0.0/RaftKeeper.tar.gz && tar -xzf RaftKeeper.tar.gz && rm RaftKeeper.tar.gz \
```

Comment:
Do not forget to update the url when you are ready to merge the pr.

Reply:
Done in most recent commit.